lib/src/sdk_api/session_service.dart

```diff
@@ -14,10 +14,15 @@ class SessionService {
 
   /// Opens a new session to connect the SDK with the Dolby.io backend. The [participantInfo] parameter refers to an additional information about the local participant.
   Future<void> open(ParticipantInfo participantInfo) async {
+    String? externalId = participantInfo.externalId;
+    if (externalId?.isEmpty == true) {
```

Comment:
What is the outcome of the if statement if `externalId` is null?

Reply:
The result of the if statement is null only if externalId is an empty string.

Reply:
Other cases are not considered

Reply:
Actually there is a bigger problem here. You are trying to solve a problem caused by the test app by changing the SDK. This is simply wrong. 

Please change the application so that it sets the `externalId` to null, and leave the SDK unchanged.

Reply:
Ok, you're right. I will fix it in test app.